compiler/compiler.go

```diff
@@ -17,6 +17,7 @@ import (
 	"google.golang.org/protobuf/types/descriptorpb"
 
 	"github.com/mercari/grpc-federation/proto/grpc/federation"
+	"github.com/mercari/grpc-federation/proto_deps/google/rpc"
```

Comment:
I thought the method of creating a `google/rpc` directory under `proto` might also work.
In this case, I thought it would be nice to just specify `-Igithub.com/mercari/grpc-federation/proto` to allow for dependency resolution.

What do you think ?